other/react-native/__tests__/__snapshots__/render-tests.js.snap

```diff
@@ -19,7 +19,6 @@ exports[`renders with React Native components 1`] = `
     onBlur={[Function]}
     onChange={[Function]}
     onChangeText={[Function]}
-    onKeyDown={[Function]}
```

Comment:
Do we not need this?

Reply:
Nope!

See https://github.com/paypal/downshift/pull/451#discussion_r215278302

Reply:
This lists the valid props for RN Text Input - no onKeyDown: https://facebook.github.io/react-native/docs/0.56/view#props